girder-dandi-archive/tests/test_rest.py

```diff
@@ -36,24 +52,24 @@ def drafts_collection(db):
 
 
 @pytest.fixture
-def dandiset_1(server, drafts_collection, user):
```

Comment:
Even if `drafts_collection` isn't being explicitly referenced, do we want to pre-create it (and the red herrings) as a precondition of this test case?

Reply:
It is used automatically on all tests in this file. See line 34: https://github.com/dandi/dandiarchive/pull/177/files#diff-1275a7650980590079021de557e6bfe4R34
```
@pytest.fixture(autouse=True)
def drafts_collection(db):
```

Reply:
Oh, I see, great.